2200002/SWEA_3499.py

```diff
@@ -0,0 +1,30 @@
+
+
+
+T = int(input())
+
+for tc in range(1, T+1):
+    N = int(input())
+    list1 = list(map(str,input().split()))
+
+    list2 = []
+    list3 = []
+
+    shuffle = int(N /2 - 0.2) 
```

Comment:
재밌는 풀이네요 😄 
```suggestion
    shuffle = int(N /2 - 0.2) 
```
이 부분을 `//` 연산을 활용하여 표현하는 방법도 고민해보시면 좋을 것 같아요!